base/ltluatex.dtx

```diff
@@ -1027,7 +1028,7 @@ local function provides_module(info)
   luatexbase_log(
     "Lua module: " .. info.name
       .. spaced(info.date)
-      .. spaced(string_gsub(info.version,"^(%d)","v%1"))
+      .. spaced(string_gsub(info.version and info.version or "","^(%d)","v%1"))
```

Comment:
If the version is not present we should not use an empty string but `nil` since otherwise `spaced` adds an redundant space.

Maybe
```suggestion
      .. spaced(info.version and string_gsub(info.version or "","^(%d)","v%1"))
```